app/src/lib/shell.ts

```diff
@@ -57,7 +57,7 @@ async function getRawShellEnv(): Promise<string | null> {
       cleanup()
     }, 5000)
 
-    const options = {
+    const options: ChildProcess.SpawnOptions = {
```

Comment:
Is there a reason this is pulled out into a constant rather than inlined directly into the `ChildProcess.spawn` call?

Reply:
i don't think so. moved! that does look nicer, thanks!